### ISSUE TITLE
[WC Payments NOX]: Design for base settings page

### ISSUE THREAD

**Issue by dmallory42** [woothemes/woocommerce#52113]
Comment taken from https://github.com/woocommerce/woocommerce/pull/52030, turned into a new issue to allow us to develop in parallel to design work:

> Thank you for working on this, @oaratovskyi! Great progress!
> 
> A couple of design inconsistencies or missing features:
> 
> 1. The whole tab content's background should be white (see this mockup):
> <img width="1440" alt="Payment Settings (Installed)" src="https://github.com/user-attachments/assets/b74263be-9987-4089-92db-02b66103a034">
> 
> 1. The payment providers list (the main list) is full-width within the container. It is not a section like the "other".
> 1. The "other" section's title is "Other payment options" not "Other payment methods"
> ![Screenshot 2024-10-15 at 15 52 15](https://github.com/user-attachments/assets/9e396c59-a722-466e-8060-ef62bae7e57f)
> 1. When collapsed, the "other" section shows the icons of the contained options
> 1. In the "other" section, the "More payment options" link has the icon at the beginning and different styling:
> ![Screenshot 2024-10-15 at 15 56 27](https://github.com/user-attachments/assets/0ff00d79-71d5-43da-971d-0b0d8ff75ef4)
> 1. The typography is not the one used in Figma (SF Pro font). We could probably go with:
> ```
> "SF Pro Text", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif
> ```
> 1. The content text's color is Gutenberg/Gray 700
> 1. The badges are styled differently (they don't have a border but a background; the radius of corners is also different):
> ![Screenshot 2024-10-15 at 16 10 16](https://github.com/user-attachments/assets/a7dcc278-7bf9-4372-a56a-f0a2137bea28)
> ![Screenshot 2024-10-15 at 16 10 21](https://github.com/user-attachments/assets/fba048ed-a27a-4ad4-a76a-a24a56c7858c)
> 1. The ellipsis menu is styled differently. We should also make sure that it is not going off-screen (I think this is an issue with this component as it exhibits the same behavior everywhere)
> ![Screenshot 2024-10-15 at 16 12 10](https://github.com/user-attachments/assets/ebf50e12-8981-4f09-8447-7682207c9f4e)
